src/schema/feeds.ts

```diff
@@ -695,21 +695,39 @@ const feedPageGeneratorWithPin: PageGenerator<
   unknown
 > = {
   connArgsToPage: ({ first, after }: FeedArgs) => {
-    const cursor = getCursorFromAfter(after);
+    const unbased = unbase64(after);
```

Comment:
Why do you need all of this?
Not really sure I get this, a simple datetime sort is enough right?

Reply:
We need the reference to the pinned date as the sorting is also used in this query, and it is required to be part of the condition for the next pages.